Sources/FaunaDB/Errors.swift

```diff
@@ -41,10 +41,16 @@ internal struct Errors {
     }
 }
 
+/// Represents an error while executing a FaunaDB query.
```

Comment:
General question: Should we add links linking back to the fauna reference docs? Or do we not do that